repltalk/__init__.py

```diff
@@ -1123,20 +1095,19 @@ async def get_user_by_id(self, user_id):
 	async def _posts_in_board(
 		self,
 		board_slugs=None,
-		languages=[],
-		order='new',
+		order='New',
```

Comment:
Please keep these lowercase to not break existing code and instead just do `.title()` in the perform_graphql.